hw1/02-palindrome.js

```diff
@@ -1,3 +1,28 @@
 const elem = document.querySelector('input');
+const resultDiv = document.querySelector('#result');
+const palindromeText = document.createElement('p');
+palindromeText.textContent = '';
+resultDiv.appendChild(palindromeText);
+
+function handleInput() {
+  if (Number.isNaN(elem.value)) {
+    palindromeText.textContent = 'Palindrome must be a number. Try again.';
+    palindromeText.style.color = 'red';
```

Comment:
Only thing  that I would recommend changing is instead of using style to add the color to the text I would suggest adding to the classList of the element so that you are using Bootstrap styling.

Reply:
Great feedback, thank you!